internal/stolonctl/pkg/crd/crd.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright (C) 2018 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package crd
+
+import (
+	"github.com/gravitational/stolon-app/internal/stolonctl/pkg/defaults"
+	"github.com/gravitational/stolon-app/internal/stolonctl/pkg/kubernetes"
+	"github.com/gravitational/stolon-app/internal/stolonctl/pkg/utils"
+
+	"github.com/gravitational/trace"
+	apiextensions "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func CreateCRD(kubeClient *kubernetes.Client) error {
```

Comment:
I would call this `CreateDefinition` or `Bootstrap` as the package is already named `crd` (and is potentially relatable to custom resource definition). If you read the point of reference:
```go
`crd.CreateCRD(...)`
```
which reads like needless repeation.